src/freenas-installer/etc/installer12.sh

```diff
@@ -1161,16 +1161,6 @@ menu_install()
     done
 
     chroot /tmp/data /usr/sbin/sysrc -f /boot/loader.conf rc_system=rc
-    chroot /tmp/data /usr/bin/touch /etc/dynamicdiskless
-    chroot /tmp/data /bin/mkdir -p /conf/base/etc
-    chroot /tmp/data /bin/sh -c '/bin/echo 65535 > /conf/base/etc/md_size'
```

Comment:
Why these should be different for EC2?

Reply:
They are the same, we are just not using installer on EC2

Reply:
We can revert these changes now, not used.

We should remove that entire file later